fortitude/resources/test/fixtures/precision/P001.f90

```diff
@@ -5,15 +5,19 @@ program test
 
   real(dp), parameter :: a = 0.0 ! Okay: No loss of precision
   real(dp), parameter :: b = 17745.0 ! Okay: No loss of precision
-  real(sp), parameter :: c = 1.0e10 ! Okay: No loss of precision, e exponent
-  real(sp), parameter :: d = 2.0E10 ! Okay: No loss of precision, E exponent
-  real(sp), parameter :: e = 1.23456 ! Bad: Loss of precision
-  real(sp), parameter :: f = 1.23456e1 ! Bad: Loss of precision, e exponent
-  real(sp), parameter :: g = 1.23456E1 ! Bad: Loss of precision, E exponent
-  real(dp), parameter :: h = 1.23456_dp ! Okay: Kind suffix
-  real(sp), parameter :: i = 1.23456_sp ! Okay: Loss of precision, but we're explicit
-  real(dp), parameter :: j = 1.23456d1 ! Okay: Ignore d exponent
-  real(dp), parameter :: k = 1.23456D3 ! Okay: Ignore D exponent
+  real(dp), parameter :: c = 0.25 ! Okay: No loss of precision
+  real(dp), parameter :: d = 0.000244140625 ! Okay: No loss of precision (=2^-12)
+  real(dp), parameter :: e = 17745.0 ! Okay: No loss of precision
```

Comment:
Same as `b`?

Reply:
Thanks, I think the suggestion might have gotten a little confused, but I removed the repetition in one of my later commits.